ios/src/TGMapViewController.mm

```diff
@@ -9,11 +9,13 @@
 
 #import "TGMapViewController.h"
 #include "platform_ios.h"
+#include "data/propertyItem.h"
 #include "tangram.h"
 
 @interface TGMapViewController ()
 
-@property (nullable, strong, nonatomic) EAGLContext *context;
+@property (nullable, nonatomic) NSString* scenePath;
```

Comment:
@blair1618 indeed bundle resolution was what I was thinking of - the `NSBundle` class provides access to the main application bundle and provides a `NSURL` set of methods and an older implementation using `NSStrings`. To be clear the string-returning functions aren't deprecated or anything. It's just common to prefer `NSURL` as it provides more safeguards and also NSURL is used by other processes to securely access bundles outside of the application sandbox for extensions. 

I see here https://github.com/tangrams/tangram-es/blob/tangram-ios-api/ios/src/platform_ios.mm#L65 that we're also using the legacy string methods for doing the bundle resolution that @karimnaaji mentioned, which looks like something I didn't manage to get to review before I went on vacation, otherwise I probably would've said the same thing there.

This is fine for now. I will probably add an extension on top of this in the Mapzen iOS SDK to add NSURLs as an option for passing along scene file locations if we want to go that route.